processing/results_analysis/gpkg_conversion.py

```diff
@@ -130,13 +129,8 @@ def processAlgorithm(self, parameters, context, feedback):
             qml_path = safe_join(styles_dir, f"{table_name}.qml")
             if os.path.exists(qml_path):
                 layer.loadNamedStyle(qml_path)
-            context.temporaryLayerStore().addMapLayer(layer)
```

Comment:
is the set "epsg_codes" on line 118 ever appended?

Reply:
Fixed.